buildconf

```diff
@@ -100,8 +100,17 @@ fi
 if [[ "$OSTYPE" == "darwin"* ]]; then
   brew=`${BREW:-brew} -v 2>/dev/null|head -n 1| sed -e 's/Homebrew^[^0-9]*//' -e 's/[a-z]* *$//'`
   if test -z "$brew"; then
-    echo "Installing Homebrew ..."
-    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    while true; do
+      read -p "Do you wish to install missing dependancis using Homebrew ? y/n `echo $'\n> '`" yn
+      case $yn in 
+        [Yy]* )
+        echo "Installing Homebrew ..."
+        /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        break;;
+        [Nn]* ) exit;;
+        * ) echo "y for Yes and n for No";
+      esac
+    done      
```

Comment:
I don't think installing the entire homebrew setup is a good idea to do from within buildconf.